### PR TITLE
Admin: Add Company's Contacts to Contact view (SHOOP-2589)

### PR DESCRIPTION
**Pull request by Pikkupomo**
Admin: Add possibility to remove contacts from company
Admin: Add possibility to add contacts for company

Refs SHOOP-2589

### CODE REVIEW

File: shoop/admin/modules/contacts/views/edit.py [hunk 1]
```diff
@@ -78,6 +82,19 @@ class Meta:  # faux ModelForm Meta
 
         self.generate_fields()
 
+    def formfield_callback(self, f, **kwargs):
+        if str(f) == "shoop.CompanyContact.members":
+            k = {
+                "initial": self.instance.members.all(),
+                "required": False,  # rules are not required by default, validation is made later on
```
Comment (tulimaki) [inline shoop/admin/modules/contacts/views/edit.py]: What rules are you talking about?

File: shoop/admin/modules/contacts/views/edit.py [hunk 2]
```diff
@@ -78,6 +82,19 @@ class Meta:  # faux ModelForm Meta
...
+                # "help_text": self.instance.description,
```
Comment (tulimaki) [inline shoop/admin/modules/contacts/views/edit.py]: Get the help_text from field?

File: shoop/admin/templates/shoop/admin/contacts/detail.jinja [hunk 3]
```diff
@@ -44,6 +44,31 @@
                 {% from "shoop/admin/macros/order_view.jinja" import order_view with context %}
                 {{ order_view(orders) }}
             {% endcall %}
+            {% if contact.members %}
+                {% call content_block(_("Members"), "fa-user") %}
+                    {% if not contact.members.count() %}
+                        {{ _("No members.") }}
```
Comment (tulimaki) [inline shoop/admin/templates/shoop/admin/contacts/detail.jinja]: Missing translation for this.

**Comment by tulimaki** [comment]
Missing tests.

File: shoop/admin/templates/shoop/admin/contacts/detail.jinja (hunk 3 above)
Comment (Pikkupomo) [inline shoop/admin/templates/shoop/admin/contacts/detail.jinja]: added

**Pull request merged**